doc/third-party/lammps-command.md

```diff
@@ -18,7 +18,10 @@ The DeePMD-kit package provides the pair_style `deepmd`
 pair_style deepmd models ... keyword value ...
 ```
 - deepmd = style of this pair_style
-- models = frozen model(s) to compute the interaction. If multiple models are provided, then the model deviation will be computed
+- models = frozen model(s) to compute the interaction. 
+If multiple models are provided, then only the first model serves to provide energy and force prediction for molecular dynamics, 
+and the model deviation will be computed among all models.
```

Comment:
The model deviation will be only computed every `out_freq` steps.

Reply:
Agree

Reply:
@njzjz plz suggest a change.